### PR TITLE
issue #181 -- combined draw/upload page

### LINKED ISSUE #181: Collapse Draw and Upload pages into one
This way, the user has one less click to make.

### PR DESCRIPTION
**Pull request by k-cleary**
Fix #181, Could needs functionality but looks pretty good. I want to add a modal for full screen muro if I have time, but this is enough to implement upload functionality
also -- user can add their own tags now (yay)

### CODE REVIEW

File: src/main/java/assets/mixxy.css [hunk 1]
```diff
@@ -18,7 +18,7 @@ a {
 h1.ui.header, h1{
     font-size: 3.75rem;
     font-family: 'Bangers', cursive;
-    margin-left: 4rem;
+    margin-left: 4rem !important;
```
Comment (JesseTG) [inline src/main/java/assets/mixxy.css]: Wait, what's this `!important` for?

File: src/main/java/www/purple/mixxy/views/ComicController/newWork.ftl.html [hunk 2]
```diff
@@ -6,25 +6,73 @@
...
-            <div class="row">
-                <div class="ui sixteen wide column comic">
-                    <div class="ui white-box">
-                        <div class="ui relaxed massive link list">
-                            <!-- need to be logged in -- if we're not logged in, redir to login page -->
-                            <div class="item">
-                                <h1><a class="create" href="/create/muro">Create a Comic with Muro</a></h1>
+        <div class="ui white-box">
+            <div class="ui fluid accordion">
+                <!-- need to be logged in -- if we're not logged in, redir to login page -->
```
Comment (JesseTG) [inline src/main/java/www/purple/mixxy/views/ComicController/newWork.ftl.html]: This logic will be implemented in one of the controllers, right?

File: src/main/java/www/purple/mixxy/views/ComicController/newWork.ftl.html [hunk 3]
```diff
@@ -6,25 +6,73 @@
...
+                            <label>Series</label>
+                            <select class="ui tags fluid search dropdown">
+                                <option value="">Series</option>
+                                <option value="1">Married with Children</option>
+                                <option value="0">Space Invaders</option>
+                            </select>
+                        </div>
+                        <div class="inline fields">
+                            <div class="required field">
+                                <select class="ui tags fluid search dropdown" multiple="">
```
Comment (JesseTG) [inline src/main/java/www/purple/mixxy/views/ComicController/newWork.ftl.html]: Wait, I thought the user would be able to enter his/her own tags?

**Comment by Bsabz** [comment]
This looks good 👍

**Pull request merged**